cli/tests/integration/mod.rs

```diff
@@ -30,6 +30,33 @@ macro_rules! itest_flaky(
 }
 );
 
+#[macro_export]
+macro_rules! itest_steps(
+($name:ident {$( $key:ident: $value:expr,)*})  => {
+  #[test]
+  fn $name() {
+    (test_util::CheckOutputIntegrationTestSteps {
+      $(
+        $key: $value,
+       )*
+      .. Default::default()
+    }).run()
+  }
+}
+);
+
+#[macro_export]
+macro_rules! command_step(
+({$( $key:ident: $value:expr,)*})  => {
+  test_util::CheckOutputIntegrationTestCommandStep {
+    $(
+      $key: $value,
+      )*
+    .. Default::default()
+  }
+}
+);
```

Comment:
Are these supposed to be included in this PR?

Reply:
Nope, forgot to remove